rust/gui-client/src-frontend/components/GeneralSettingsPage.tsx

```diff
@@ -70,56 +70,61 @@ export default function GeneralSettingsPage({
           />
         </div>
 
-        <div className="flex flex-row gap-2 items-center">
-          <ToggleSwitch
-            name="start_minimized"
-            id="start-minimized-input"
-            checked={localSettings.start_minimized}
-            onChange={(e) =>
-              setLocalSettings({
-                ...localSettings,
-                start_minimized: e,
-              })
-            }
-          />
-          <Label className="text-neutral-600" htmlFor="start-minimized-input">
-            Start minimized
-          </Label>
-        </div>
+        <div className="max-w-1/2">
+          <div className="mt-4 flex justify-between items-center">
```

Comment:
Why `mt` instead of `gap`? gap is much more elegant.

Reply:
fixed